schemas/cslc_s1.yaml

```diff
@@ -0,0 +1,74 @@
+runconfig:
+    name: str()
+
+    groups:
+        pge_name_group:
+           pge_name: enum('CSLC_S1_PGE')
+
+        input_file_group:
+           # Required. List of SAFE files (min=1)
+           safe_file_path: list(str(), min=1)
+           # Required. List of orbit (EOF) files
+           orbit_file_path: list(str(), min=1)
```

Comment:
What's the use case for more than 1 `orbit_file_path`?

Reply:
We have specified the `safe_file_path` as a list of file paths to SAFE files (e.g. more than one date). If this is correct, you want to assume that you will have more orbit files. Am I missing something?

Reply:
I forgot we're not supplying the orbit file(s). My bad.